.github/workflows/ci-wheels.yml

```diff
@@ -0,0 +1,120 @@
+# Reference:
+#   - https://github.com/actions/checkout
+#   - https://github.com/actions/download-artifact
+#   - https://github.com/actions/upload-artifact
+#   - https://github.com/pypa/cibuildwheel
+#   - https://github.com/pypa/build
+#   - https://github.com/pypa/gh-action-pypi-publish
+#   - https://test.pypi.org/help/#apitoken
+
+name: ci-wheels
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - "main"
+      - "v*x"
+      - "!auto-update-lockfiles"
+      - "!pre-commit-ci-update-config"
+      - "!dependabot/*"
+    tags:
+      - "v*"
+
+jobs:
+  build_bdist:
+    name: "build ${{ matrix.os }} (${{ matrix.arch }}) wheels"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
It looks like Tephi is agnostic to OS - there are no OS-specific files here:

https://pypi.org/project/tephi/#files [^1]

So the matrix should be unnecessary. [E.g. `ci-wheels.yml` in Iris does not have one](https://github.com/SciTools/iris/blob/2443396fc18aa17dad1d0461fb6e885283ed7b51/.github/workflows/ci-wheels.yml#L27-L43).

[^1]: [Compare this to stratify, which has many OS-specific files](https://pypi.org/project/stratify/#files).